### ISSUE TITLE
Add guide for using Mizuki CFW (https://github.com/thedax/Mizuki/releases) for Step 5 and for region changing

### ISSUE THREAD

**Issue by jaxrogers2** [Plailect/Guide#45]
Since this guide now uses an updated sysNAND format for A9LH, and this CFW (Mizuki) was just released specifically for this setup, it might be a good addition to have a guide for arm9loaderhax (step 5) with this CFW along with AuReiNAND and CakesFW. The CFW uses arm9 select by FIX94 for loading applications on boot (including itself) and is the first A9LH CFW that supports the 3DS loader replacement by yifanlu (https://github.com/yifanlu/3ds_injector). This loader then adds more features, such as region free loading from home menu and even region changing support. This CFW is minimalist CFW, so only the CFW's arm9loaderhax.bin is included in the download. In order to run, the AuiReiNAND firmware.bin files can be used and must be put in a /mizuki folder on the SD card, along with the custom loader, which must be named injector.cxi. From there, the arm9loaderhax.bin included in the Mizuki download must be renamed to default.bin and saved to the /arm9select folder on the SD card. For loading other applications on boot, the .bin files (such as decrypt9WIP.bin) have to be renamed to the proper button to be held on boot (a.bin, b.bin, etc) and put in the /arm9select folder, like in the AuReiNAND guide.

This CFW is probably the best option for beginners because it's less complicated compared to AuReiNAND and Cakes yet it supports nearly all of their features, including region free and loading apps from boot. For the guide, the hblauncher_loader and the A9LH installer files would still be needed (safeA9LH installer, slotkey bins, payload_input.zip, minipasta, etc.) along with the "on boot" applications (such as Decrypt9WIP and uncart for A9LH). However, other than that, all that would be needed is the most recent release of Mizuki, the 3DS custom loader, arm9select, and the AuReiNAND NTR firmware.bin files for Old and New3DS.

Also, this CFW can be used to switch regions on A9LH with Eshop access due to the custom loader, so that functionality could be utilized in the region changing guide.

**Comment by jaxrogers2** [comment]
Also, I forgot about something- the loader supports region changing eshop too!

**Comment by Plailect** [comment]
I have decided against adding this CFW as an option (for now) because of the following reasons:

+ 2 Options is already more than I initially wanted, I'd prefer no more
+ Arm9select similar functionality is already integrated into AuReiNAND
+ This is not the first CFW to support 3ds_injector, AuRei has had it integrated for days
+ "Minimalist" means nothing, and AuRei *only* need the arm9loaderhax.bin file to work, the rest aren't used other than the empty aurei folder
+ It's not less complicated than AuRei (other than not needing options turned on/off), and doesn't seem to offer any extra features in comparison
+ Finally, the actual region changing steps for yifanlu's eshop stuff is [far more complicated than just using his loader](https://gist.github.com/yifanlu/e80db121d38aceb8cca0e03cefd5853b)

**Comment by jaxrogers2** [comment]
Thanks for clarifying what the custom loader does, I really didn't know too much about it before. I think the main reason I suggested this is simply because I should probably use AuiReiNAND for my CFW, it's just that right now it's being updated very frequently and I just don't want to redownload my CFW that often. Before using Mazuki CFW, I used Cakes because AuReiNAND had not yet came out yet when I installed A9LH. But I didn't like the bootCTR9 setup, which was slower than arm9select and the Smash 4/MH4 reboot patch in Cakes broke GBA/DSiWare games. While Cakes requires less in terms of folder structure compared to AuReiNAND, it's still a decent amount of setup. Mazuki CFW is just a good mixture of both- sure, it's not made to add any new features, it's really there to condense them down. That way, when this CFW is updated, there might be only a couple of files to replace, in comparison to many files when an AuiReiNAND or Cakes update occurs. Also, it gives me the two things a CFW on A9LH needs to provide: Booting applications with permissions and patching reboots and firmware without any extra clutter or even a menu. Those are just the reasons for my Mazuki CFW setup, if people want to use AuiReiNAND or Cakes instead though I don't really care what you choose.
